Celeste.Mod.mm/Content/Dialog/English.txt

```diff
@@ -33,6 +33,9 @@
 	MENU_MODOPTIONS_MULTIPLE_MODS_FAILEDTOLOAD=	{0} mods failed to load
 	MENU_MODOPTIONS_UPDATE_AVAILABLE=			An Everest update is available
 
+# File Select Naming
+	NAME_HINT_KEYBOARDINPUT=You're in keyboard input mode.
+	NAME_HINT_USECONTROLLER=Press A on a controller to switch to vanilla mode.
```

Comment:
Please remove these lines if they're unnecessary for this PR.

Reply:
Alright, they're removed now